build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Build steps to validate the ketohub raw input spider.
+
+# Run unit tests and calculate code coverage.
+coverage run \
+  --source ketohub \
+  -m unittest discover &&
+# Check that source has correct formatting.
+yapf --diff --recursive --style google ketohub/spiders/*.py &&
```

Comment:
We should be good. If I'm reading this right, they released a new version Sunday: 

https://github.com/google/yapf/blob/master/CHANGELOG#L15